### PR TITLE
Remove white outlines from buttons

### LINKED ISSUE #1349: Remove white outline from 'SEE MORE' and 'LOAD MORE' buttons on One Live pages
**Describe the bug**
There is a design breaking white outline on the 'SEE MORE' and 'LOAD MORE' buttons

**Expected behavior**
Should not have the white outlines on buttons

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by miniskynet**
## Purpose

The purpose of this PR is to fix #1349 

## Goals

Remove design breaking white outlines from 'SEE MORE' and 'LOAD MORE' buttons on One Live pages

## Approach

Remove the class btn-outline-white and replace it with border-0

### Screenshots

**Now:**
image
image

### Preview Link

https://pr-1350-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by miniskynet** [comment]
@anjula-sack @Piumal1999 I noticed a color change on button click even however. It turns to blue when clicked. Not sure if that should be kept

File: onelive/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" rel="stylesheet">
     <!-- Argon CSS -->
     <link type="text/css" href="../assets/css/argon.css?v=1.0.1" rel="stylesheet">
+    <link type="text/css" href="../assets/css/argon.css?v=1.0.1" rel="stylesheet">
```
Comment (anjula-sack) [inline onelive/index.html]: Can you remove this duplicated line?

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: onelive/index.html (hunk 1 above)
Comment (miniskynet) [reply onelive/index.html]: Done. Seems fine?

**Comment by anjula-sack** [comment]
@miniskynet can you make this PR ready for review?

**Review by anjula-sack — APPROVED** [review]
Thank you! @miniskynet

**Pull request merged**